django/chat/templates/chat/components/chat_history_sidebar.html

```diff
@@ -31,12 +31,13 @@
     </div>
     <div class="p-3">
       <button type="button"
-              class="btn btn-outline-danger btn-outline-initially-black w-100"
+              class="btn btn-outline-danger btn-outline-initially-black w-100 d-flex justify-content-center align-items-center"
```

Comment:
With the longer French translation, I needed to center and align the button. Didn't spend too much time on it because I'm not 100% sure this is the translation we want to use.